README.md

```diff
@@ -225,6 +225,10 @@ ts2gd generates code with 2 spaces as indent. If Godot keeps changing your .gd f
 - Goto Editor -> Editor Settings -> Text Editor -> Indent
 - And switch to Type: Spaces and Size: 2
 
+## Ignoring sub directories containing TypeScript files
+
+In case your repository has further TypeScript files, we shouldn't be transpiled via ts2gd, you can add an ignore entry to your ts2gd.json configuration file. That entry takes an array of [anymatch](https://www.npmjs.com/package/anymatch) strings or globs.
```

Comment:
Two writing nits:

    I would rephase this like so: "If you would like to tell ts2gd to ignore certain TypeScript files, you can add "ignore": /* list of files */ to your ts2gd.json file.
    I would give a handful of common examples rather than referring the user to anymatch. For instance, a list of 1) how to ignore a file, 2) how to ignore two different files, and 3) how to ignore everything inside a folder. After those 3 items, you could include a link to anymatch if they still need help.